README.md

```diff
@@ -428,17 +281,15 @@ Chapter 14, page 328, "Ensure Unambiguous Provenance." Instead, we recommend
 keying primarily by resource name, in addition to content hash.
 
 [Binary Authorization]: https://cloud.google.com/binary-authorization
-[DigestSet]: spec/field_types.md#DigestSet
-[Envelope]: #envelope
-[ITE-5]: https://github.com/MarkLodato/ITE/blob/ite-5/ITE/5/README.md
+[Envelope]: spec/envelope.md
 [ITE-6]: https://github.com/in-toto/ITE/pull/15
 [Link]: spec/predicates/link.md
-[Predicate]: #predicate
+[Predicate]: spec/predicate.md
 [Provenance]: spec/predicates/provenance.md
-[RFC 3339]: https://tools.ietf.org/html/rfc3339
-[RFC 3986]: https://tools.ietf.org/html/rfc3986
 [SLSA Attestation Model]: https://github.com/slsa-framework/slsa-controls/blob/main/attestations.md
 [SLSA]: https://github.com/slsa-framework/slsa
-[Statement]: #statement
+[SPDX]: spec/predicates/spdx.md
+[Statement]: spec/statement.md
```

Comment:
Should this and 'Predicate' link to spec/README.md?

Reply:
Oops, yup! Fixed.